src/sig/falcon/pqclean_falcon-1024-padded_aarch64/LICENSE

```diff
@@ -0,0 +1,57 @@
+This ARMv8 NEON implementation is provided under the Apache 2.0 license:
+
+/*
+ * =============================================================================
+ * Copyright (c) 2023 by Cryptographic Engineering Research Group (CERG)
+ * ECE Department, George Mason University
+ * Fairfax, VA, U.S.A.
+ * Author: Duc Tri Nguyen
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Hmm -- Falcon right now is listed as "MIT" (only) if I'm not mistaken(?). Doesn't this new LICENSE text (adding Apache2) warrant a change to this (at least in top level doc .yml/.md)? Are the SPDX statements in line?

Reply:
Good catch. The existing Falcon `aarch64` code is also by @ducnguyen-sb and also uses the Apache2 licence, so it should be updated as well.

Reply:
Hi, 

I don't mind changing the License. I remember back when I develop aarch64, it was Apache 2.0, so I set mine as Apache 2.0. 
If you want MIT, I can make it instantly.

Reply:
I honestly don't mind. Whatever is easiest to bring LICENSE, documentation and SPDX in sync.

Reply:
I've updated the documentation to clarify that the `aarch64` implementation follows the Apache 2.0 licence.

Reply:
Just remembered that we don't expect the "upstream" files to have SPDX statements in their files. The above then resolves this comment completely. Thanks also for this, @SWilson4